changelog-2023.09.22-27.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index cb6033f925..17a6cf3064 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -113,6 +113,27 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2023.09.22-27" date="2023-09-22">
++     <description>
++       <p>Highlights:</p>
```

Comment:
• New OSM maps data as of September 20
• Add positioning support via GeoClue2
• Use OpenGL ES 3.0 instead of desktop OpenGL
• Added new route type "Ruler" (aka Helicopter Routing) to measure straight distances quickly
• Set decimal and grouping separators depending on the system locale
• Caves can be added in the Editor

Search
• Parking entrances are searchable
• Improved street matching and results relevance
• Search for grocery and houseware

Translations:
• Improved Arabic, Polish, and Spanish translations

Map styling:
• New or updated icons for factory, winery, bookcase, library, brothel, bicycle repair station, bicycle rental, bicycle parking, sauna, lottery, bookmaker, massage, sewing, books, underground parking, disabled parking space, gate, excrement bags vending, wholesale shop, outdoor seating, handicraft, grinding mill, caterer, HVAC, key cutter and locksmith
• Added vehicle inspection, public bath, strip club, gambling, cycle barrier, waterways (drain, ditch, moat, wastewater), aquarium, assembly point, adult gaming center, amusement arcade, Mormon churches, visitor center, industrial mine, historical objects (cannon, anchor, stone, aircraft, tank, mine, wreck, locomotive)
• Added bridges and tunnels rendering for cycleways, footways, paths, tracks, bridleways, steps
• Render barrier=ditch differently from waterway=ditch
• Reduced priorities of river names (and of other lower-importance lines) in navigation mode
• Tuned road names and road shield priorities
• Fixed icons display for parking buildings
• Reduced visibility of minor and private parking icons, removed area fill for underground parking
• Reduced fire hydrant visibility
• Display house numbers above icons and main POI captions, and don’t display house numbers for buildings that are too small for the current zoom level
• Don't display star icons for squares
• Fine-tune house numbers font and drawing priority
• Added highway-pedestrian, bridleway, and cycleway lines for detailed zooms (z18-) in navigation mode
• Pedestrian tunnels are not visible in navigation mode